README.md

```diff
@@ -33,52 +33,92 @@ in the x-common repository.  This describes how all the language tracks are put
 the common metadata, and high-level information about contributing to existing problems and adding new problems.
 In particular, please read the [Pull Request Guidelines](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#pull-request-guidelines) before opening a pull request.
 
-## Problem Versioning
-
-Each problem defines a `const targetTestVersion` in the test program, and validates that the solution has defined a matching value `testVersion`.  Any xgo developer that changes the test program or test data increments `targetTestVersion`.
-
-The benefit of all this is that reviewers can see which test version a posted solution was written for and be spared confusion over why an old posted solution might not pass current tests.
-
-Notice that neither the `testVersion` nor the `targetTestVersion` is exported. This is so that golint will not complain about a missing comment. In general, adding tests for unexported names is considered an anti-pattern, but in this case the trade-off seems acceptable.
-
 ## Xgo style
 
-Let's walk through the first problem in `config.json`, leap.  Cd down into the leap directory now, there are two
```

Comment:
ha, oops... only just realized this has been out of date since #206 - over a year. Better late than never.